plutus-metatheory/src/Algorithmic/Completeness.lagda

```diff
@@ -74,12 +74,14 @@ lem[] A B = trans
 
 
 nfTypeTC : ∀{φ}{A : φ ⊢⋆ *} → STermCon.TermCon A → NTermCon.TermCon (nf A)
-nfTypeTC (STermCon.integer i)    = NTermCon.integer i
-nfTypeTC (STermCon.bytestring b) = NTermCon.bytestring b
-nfTypeTC (STermCon.string s)     = NTermCon.string s
-nfTypeTC (STermCon.bool b)       = NTermCon.bool b
-nfTypeTC STermCon.unit           = NTermCon.unit
-nfTypeTC (STermCon.pdata d)       = NTermCon.pdata d
+nfTypeTC (STermCon.tmInteger i)    = NTermCon.tmInteger i
+nfTypeTC (STermCon.tmBytestring b) = NTermCon.tmBytestring b
+nfTypeTC (STermCon.tmString s)     = NTermCon.tmString s
+nfTypeTC (STermCon.tmBool b)       = NTermCon.tmBool b
+nfTypeTC STermCon.tmUnit           = NTermCon.tmUnit
+nfTypeTC (STermCon.tmData d)       = NTermCon.tmData d
+--nfTypeTC (STermCon.tmPair x y)     = NTermCon.tmPair (nfTypeTC x) (nfTypeTC y)
+--nfTypeTC (STermCon.tmList xs)      = NTermCon.tmList (map nfTypeTC xs)
```

Comment:
Those are for the typed term constant, which I thought that I needed to do, then I realised that I could throw an error when typecheking lists. I left them commented because I will have to write them at some point.

Reply:
deleted